templates/deployment.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
Shouldn't we use a statefulset instead of deployment? I think about predictable name is almost mandatory if you scale it and needs to have a consistent sharing between queues.

Reply:
So we should make queues more configurable